src/metrics/coefficientofdetermination.jl

```diff
@@ -21,11 +21,11 @@ function R2coefficientofdetermination(
     return R2
 end
 
-function fractionofvarianceunexplained(
+function fraction_of_variance_unexplained(
```

Comment:
changed to variance_explained() ... leave the format (fraction) to be explained in the documentation

Reply:
I might just remove this function. FVU is defined as 1- R^2, so it's trivial to compute. And frankly I never use it.

Reply:
Done